Apps/Medication/src/Models/FederalDrugSource.cs

```diff
@@ -31,6 +31,6 @@ public class FederalDrugSource
         /// <summary>
         /// Gets or sets the drug product.
         /// </summary>
-        public DrugProduct DrugProduct { get; set; }
+        public DrugProduct? DrugProduct { get; set; }
```

Comment:
I dont think it makes sense to have the DrugProduct as nullable if its parent object exist at all. It would be a date holder without it.

Reply:
btw, this is the only one that is a _show stopper_ for me

Reply:
I can change this, the only reason I set it to nullable was because we have to initialize it to an empty Drug which we throw away.   Its wasted Object allocation but I've added it.